darts/tests/models/forecasting/test_sklearn_models.py

```diff
@@ -1098,7 +1098,9 @@ def test_fit(self, config):
         # auto-regression but past_covariates does not extend enough in the future
         with pytest.raises(ValueError):
             model_instance = model(lags=4, lags_past_covariates=4, multi_models=mode)
-            model_instance.fit(series=series, past_covariates=sine_multivariate1)
+            model_instance.fit(
```

Comment:
let's revert this change, since we want to check for a specific error, and verbose should not be part of it